lambdas/process-async-cri-credential/src/main/java/uk/gov/di/ipv/core/processasynccricredential/ProcessAsyncCriCredentialHandler.java

```diff
@@ -114,10 +116,19 @@ public SQSBatchResponse handleRequest(SQSEvent event, Context context) {
                     | SqsException
                     | CiPutException
                     | AsyncVerifiableCredentialException
+                    | UnrecognisedVotException
                     | CiPostMitigationsException e) {
                 LOGGER.error(
                         LogHelper.buildErrorMessage("Failed to process VC response message.", e));
                 failedRecords.add(new SQSBatchResponse.BatchItemFailure(message.getMessageId()));
+            } catch (AuditExtensionException e) {
+                LOGGER.error(
+                        new StringMapMessage()
+                                .with(
+                                        LOG_MESSAGE_DESCRIPTION.getFieldName(),
+                                        "Failed to process/send audit event.")
+                                .with(LOG_ERROR_DESCRIPTION.getFieldName(), e.getMessage()));
```

Comment:
```suggestion
                LOGGER.error(LogHelper.buildErrorMesssage("Failed to process/send audit event.", e.getMessage()));
```

I think we can use the above which is functionaly equivalent and a bit less verbose.